### PR TITLE
Update installation page and delete some out-dated descriptions

### PR DESCRIPTION
**Pull request by yfei1**

### CODE REVIEW

File: site/content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ description: >
   Follow this guide to setup Open Match in your Kubernetes cluster.
 ---
 
-In this quickstart, we'll create a Kubernetes cluster, install Open Match, and create matches with the example tools.
+In this quickstart, we'll create a Kubernetes cluster and install Open Match.
 
 ## Setup Kubernetes Cluster
 
-This guide is for users that do not have a Kubernetes cluster. If you already have one that you can install Open Match into, skip this section.
+This guide is for users that do not have a Kubernetes cluster. If you already have one that you can install Open Match into, see [how to get your first match with Open Match]({{< ref "/docs/Getting Started" >}}).
```
Comment (Laremere) [inline site/content/en/docs/Installation/_index.md]: What this comment is saying is that you can skip the setup k8s cluster section (note the number of #s, more means a nested section under fewer).  They still need to install open match itself.

File: site/content/en/docs/Installation/_index.md [hunk 2]
```diff
@@ -46,48 +46,82 @@ gcloud container clusters create --machine-type n1-standard-2 open-match-dev-clu
...
+Open Match comprises of a set of services that enable core functionality such as Ticket Management and Filtering etc. It also includes a
+state storage to persist state needed for its functioning.
 
 The simplest way to install Open Match is to use the install.yaml files for the latest release.
 This installs Open Match with the default configuration.
 
 ```bash
 # Create a cluster role binding (if using gcloud on Linux or OSX)
-kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user `gcloud config get-value account`
+> kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user `gcloud config get-value account`
```
Comment (Laremere) [inline site/content/en/docs/Installation/_index.md]: Why these brackets?

It removes the ability to copy paste a block of commands and run them.

File: site/content/en/docs/Installation/_index.md [hunk 3]
```diff
@@ -46,48 +46,82 @@ gcloud container clusters create --machine-type n1-standard-2 open-match-dev-clu
...
+  - A Redis deployment as Open Match's state storage system.
+  - ServiceAccounts, Roles and RoleBindings to define Open Match deployments' IAMs. 
+  - Several Services for Open Match endpoints. 
+  - HorizontalAutoScalars to auto-scale Open Match based on pods' average CPU utilization. 
 
-- Installing a match making function and evaluator.
-- Updating the configmap with the correct indexes.
-
-[Getting Started]({{< ref "/docs/Getting Started" >}}) contains an example which covers this.
+```
```
Comment (Laremere) [inline site/content/en/docs/Installation/_index.md]: What's this section for?  This is likely to change a lot and not be of much use?

Comment (yfei1) [reply site/content/en/docs/Installation/_index.md]: I added this section to address Mark's concern. He pointed out that

>  Wow this is a lot of things installed. Would be good to have a component list somewhere.
Why do I have cassandra in here?

I truncated this list a bit to make it less redundant.

Comment (Laremere) [reply site/content/en/docs/Installation/_index.md]: I think the section is reasonable, but the actual list of things created isn't.  Lots of information with no context or explanation is just noise.

Comment (yfei1) [reply site/content/en/docs/Installation/_index.md]: Deleted.

**Pull request merged**